### PR TITLE
Make price function configurable

### PR DESCRIPTION
**Pull request by berndbohmeier**
this should make the auction better testable

### CODE REVIEW

File: contracts/contracts/ValidatorAuction.sol [hunk 1]
```diff
@@ -13,7 +13,8 @@ contract ValidatorAuction is Ownable {
 
     uint public startTime;
     uint public closeTime;
-    uint public constant AUCTION_DURATION = 14 days;
+    uint public auctionDurationInDays;
+    uint public startPrice;
     uint public constant NUMBER_OF_PARTICIPANTS = 123;
```
Comment (cducrest) [inline contracts/contracts/ValidatorAuction.sol]: Might as well parametrise the number of participants

File: contracts/tests/conftest.py [hunk 2]
```diff
@@ -25,6 +25,9 @@
 MALICIOUS_VALIDATOR_INDEX = HONEST_VALIDATOR_COUNT
 MALICIOUS_NON_VALIDATOR_INDEX = MALICIOUS_VALIDATOR_INDEX + 1
 
+AUCTION_DURATION_IN_DAYS = 14
+START_PRICE_IN_ETH = 10000
```
Comment (cducrest) [inline contracts/tests/conftest.py]: AUCTION_START_PRICE_IN_ETH

**Comment by andfletcher** [comment]
https://github.com/trustlines-network/project/issues/319

File: contracts/contracts/ValidatorAuction.sol (hunk 1 above)
Comment (berndbohmeier) [reply contracts/contracts/ValidatorAuction.sol]: yeah, but I thought we dont need it really. Do you think I should do it?

File: contracts/tests/conftest.py (hunk 2 above)
Comment (berndbohmeier) [reply contracts/tests/conftest.py]: done

File: contracts/contracts/ValidatorAuction.sol (hunk 1 above)
Comment (berndbohmeier) [reply contracts/contracts/ValidatorAuction.sol]: okay

File: contracts/tests/test_validator_auction.py [hunk 3]
```diff
@@ -346,7 +360,7 @@ def test_event_whitelist(no_whitelist_validator_auction_contract, whitelist, web
 
 
 ETH_IN_WEI = 1e18
-START_PRICE = 10000 * ETH_IN_WEI
+AUCTION_START_PRICE = 10000 * ETH_IN_WEI
```
Comment (cducrest) [inline contracts/tests/test_validator_auction.py]: Should this not be at the beginning of the file?
Also you have duplication of `AUCTION_START_PRICE`

Comment (berndbohmeier) [reply contracts/tests/test_validator_auction.py]: Fixed

**Pull request merged**